### PR TITLE
Replace Aurora with Horizon (dynamically), NT -> SCC

### PR DESCRIPTION
**Pull request by JohnWildkins**
title
almost every reference to the Aurora (that isn't meant to specifically reference the Aurora, like the minatures) is replaced with a dynamic station name that will display Horizon, Aurora, or Runtime depending
some references to NanoTrasen were changed to SCC (and referencing 'station' was changed to facility or some other neutral term), where deemed appropriate.

I'm not the best judge of this stuff and I'm not a super in-tune lore nerd, so if I screwed something up lemme know.

### CODE REVIEW

**Comment by JohnWildkins** [comment]
!review

File: code/controllers/subsystems/job.dm [hunk 1]
```diff
@@ -457,7 +457,7 @@
 
 	var/datum/spawnpoint/spawnpos = SSatlas.spawn_locations["Cryogenic Storage"]
 	if(spawnpos && istype(spawnpos))
-		to_chat(src, "<span class='warning'>You come to the sudden realization that you never left the Aurora at all! You were in cryo the whole time!</span>")
+		to_chat(src, "<span class='warning'>You come to the sudden realization that you never left the [current_map.station_name] at all! You were in cryo the whole time!</span>")
```
Comment (alsoandanswer) [inline code/controllers/subsystems/job.dm:460]: cryo doesn't exist in the horizon, i think. been replaced with living quarters.

File: code/game/antagonist/station/revolutionary.dm [hunk 2]
```diff
@@ -33,6 +32,7 @@ var/datum/antagonist/revolutionary/revs
 	required_age = 31
 
 /datum/antagonist/revolutionary/New()
+	faction_welcome = "You joined a subversive organization in the crew of the [current_map.station_name]. United under a forward-thinking leader, you must achieve their goals."
```
Comment (alsoandanswer) [inline code/game/antagonist/station/revolutionary.dm:35]: it's runtiming here. current_map is null here.

**Review by alsoandanswer — CHANGES_REQUESTED** [review]

File: code/controllers/subsystems/job.dm (hunk 1 above)
Comment (Caelphon) [reply code/controllers/subsystems/job.dm:460]: cryo + living quarters exist

File: code/game/antagonist/station/revolutionary.dm [hunk 3]
```diff
@@ -7,7 +7,7 @@ var/datum/antagonist/revolutionary/revs
 	bantype = "revolutionary"
 	feedback_tag = "rev_objective"
 	antag_indicator = "contenderhead"
-	welcome_text = "You are a subversive seeking to demolish the current order on the station by whatever means possible. Recruit friends and strangers alike to bring Nanotrasen's tyranny to an end! Or whatever your objective is."
+	welcome_text = "You are a subversive seeking to demolish the current order on the station by whatever means possible. Recruit friends and strangers alike to bring the SCC's tyranny to an end! Or whatever your objective is."
```
Comment (mikomyazaki) [inline code/game/antagonist/station/revolutionary.dm:10]: Bunch of the `welcome_text` ,  `extended_round_description` , `round_description`  can be moved to `New` or `Initialize` and use the map vars.

File: code/game/jobs/faction/faction.dm [hunk 4]
```diff
@@ -69,12 +69,12 @@
 	switch(mission_level)
 		if(REPRESENTATIVE_MISSION_HIGH)
 			objective = pick("Assist your contractor in smuggling [rand(1,4)] items of value",
-							"Collect evidence of Nanotrasen being unfair or oppressive against your contractors to be used as leverage in future talks")
+							"Collect evidence of the SCC being unfair or oppressive against your contractors to be used as leverage in future talks")
```
Comment (mikomyazaki) [inline code/game/jobs/faction/faction.dm:72]: Can use `current_map.company_short` here and elsewhere.

**Review by mikomyazaki — CHANGES_REQUESTED** [review]

File: code/controllers/subsystems/job.dm (hunk 1 above)
Comment (JohnWildkins) [reply code/controllers/subsystems/job.dm:460]: doesn't really matter if it exists or not, it should be dynamic regardless

File: code/game/jobs/faction/faction.dm (hunk 4 above)
Comment (JohnWildkins) [reply code/game/jobs/faction/faction.dm:72]: replaced where I could, but I didn't really want to override New() 100 times just to make it dynamic for this specifically

**Review by alsoandanswer — APPROVED** [review]

**Review by mikomyazaki — APPROVED** [review]

**Pull request merged**